Use get_gpu_type to detect ivshmem device

phreer opened the pull request.

phreer committed:
Use get_gpu_type to detect ivshmem device

Tracked-On: OAM-127906
Signed-off-by: Weifeng Liu <weifeng.liu@intel.com>

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

phreer committed:
Use ivshmem-backed virtio-GPU as KMS driver in the case of QNX

The QNX scenario uses ivshmem-backed virtio-GPU for display, a special
use case for us. We distinguish this scenario by checking whether the
ivshmem virtio-GPU is the only virtio-GPU on the system.

Tracked-On: OAM-127906
Signed-off-by: Weifeng Liu <weifeng.liu@intel.com>

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

feijiang1 reviewed (approved):
LGTM

feijiang1 reviewed (approved):
LGTM

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon_umr0_master-merge/282
